buzzmobile/sense/lidar/lidar.py

```diff
@@ -45,9 +76,26 @@ def genLidarImage(lidarPoints):
 
     #cv2.fillConvexPoly(matrix, np.array(polyPoints, np.int32), 0)
     cv2.fillPoly(matrix, [np.array(polyPoints, np.int32)], 0)
-    cv2.imshow("yo", matrix)
+    '''cv2.imshow("yo", matrix)
     cv2.waitKey(0)
-    cv2.destroyAllWindows()    
+    cv2.destroyAllWindows()'''
+    return matrix
+
+
+#genLidarImage([(1, 2.4), (2, 3.3), (1.5, 9), (3.5, 3.6)])
+
+def lidar_node():
+    #only you can prevent null pointers
+    pub.publish('')
+    rospy.init_node('lidar_node', anonymous=True)
+    rospy.Subscribe('scan', LaserScan, gen_lidar_image, queue_size=0)
+    rospy.spin()
+
+if __name__ == '__main__': lidar_node()
```

Comment:
I was just going by the fact that __main__ was a one liner in framer_merger as well and the two files should be somewhat stylistically consistent